asv_bench/benchmarks/arithmetic.py

```diff
@@ -469,4 +469,26 @@ def time_apply_index(self, offset):
         offset.apply_index(self.rng)
 
 
+class SubMultiIndex:
```

Comment:
can you rename to BinaryOpsMultiIndex, and parametrize over sub,add,mult,div

Reply:
Done, thx.